acceptance/.gitignore

```diff
@@ -1,3 +1,5 @@
+# Do not add environment-specific entries here (see the top-level .gitignore
+# for reasoning and alternatives).
```

Comment:
The reasoning is at the beginning of the comment:

# Editor or environment-specific entries should not be committed to the
# repository but can be kept in git's global configuration:
#
# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

An alternative case could be made, but some issues aren't worth the time
invested in them and I personally will be happy not to see any more random
PRs that add this or that file someone chanced upon on their system - at
least now it's clear what to do.

On Thu, Jul 21, 2016 at 4:25 PM Tristan Rice <notifications@github.com>
wrote:

> In acceptance/.gitignore
> <https://github.com/cockroachdb/cockroach/pull/7956#discussion_r71780365>:
>
> > @@ -1,3 +1,5 @@
> > +# Do not add environment-specific entries here (see the top-level .gitignore
> > +# for reasoning and alternatives).
>
> The top level .gitignore doesn't have the reasoning for this change, just
> how to work around it. Is having extra ignores an issue?
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7956/files/cfe229ccf2b931a6b9dee4f44f3b12e09abbbe1a#r71780365>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135FG0fEvef9LmO2TOjFNb__d1jhvGks5qX9WzgaJpZM4JR2F6>
> .
>
-- 

-- Tobias